stdlib/2and3/linecache.pyi

```diff
@@ -0,0 +1,13 @@
+import sys
+from typing import Any, Dict, List, Optional, Text, Union
+
+_ModuleGlobals = Dict[str, Any]
+_FileName = Union[str, Text]
```

Comment:
Oh yes, I should have just written Text. Py3 linecache doesn't support bytes paths.